src/hunt-intake.php

```diff
@@ -302,6 +302,17 @@
             $processed_details['pillage_type'] = True;
             $processed_details['pillage_amount'] = True;
         }
+        # Process details for the location
+        if (file_exists("locations/locations.php")) {
+            include("locations/locations.php");
+            $loc_id = filter_var($_POST['location']['id'], FILTER_VALIDATE_INT);
+            if (array_key_exists($loc_id, $location_id_to_filter_name)) {
+                $loc_file = "locations/" . $location_id_to_filter_name[$loc_id] . ".php";
+                if (file_exists($loc_file)) {
+                    include($loc_file);
+                }
+            }
+        }
```

Comment:
```suggestion
        include_once "locations/locations.php";
```
and then inside the locations.php file i'd just do:
```
$loc_id = $_POST['location']['id'];
if (is_numeric($loc_id) && array_key_exists($loc_id, $location_id_to_filter_name) && file_exists("locations/" . $location_id_to_filter_name[$loc_id] . ".php")) {
    include_once "locations/" . $location_id_to_filter_name[$loc_id] . ".php";
}
```